### PR TITLE
Portfolio: Desktop Version

### PR DESCRIPTION
**Pull request by ndikumanaisaie**
This branch contains the desktop version which is responsive to many devices, smart phones of different size, tablets and big screens.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -239,30 +229,42 @@ <h2 class="h2">About Myself</h2>
...
+            <li class="language-list">
+              <ul>
+                <li><a href="#"> <img src="./images/js.png" alt=""><span class="js">JavaScript</span> </a></li>
+                <li><a href="#"><img src="./images/html.png" alt=""><span>HTML</span></a></li>
+                <li><a href="#"><img src="./images/css.png" alt=""><span>CSS</span></a></li>
+              </ul>
+            </li>
+            <li class="my-work">
+              <span >Frameworks</span>
+              <span class="greater-than"><i class="fa-solid fa-angle-right"></i></span>
```
Comment (sja-thedude) [inline index.html:241]: Desktop Version:
- [ ] Kindly follow spacing for the About myself section by giving more spacing to the left side as per the Figma design.
- [ ] Kindly follow the font weight for the list as shown in the attached video.
- [ ] Kindly reduce the space below the "About Myself" title as per the figma design.

https://user-images.githubusercontent.com/70269507/165737871-b685c2b5-3c9e-41cc-a6c2-c3d742e61482.mov

File: index.html [hunk 2]
```diff
@@ -239,30 +229,42 @@ <h2 class="h2">About Myself</h2>
...
-              </div>
+            <li class="language-list">
+              <ul>
+                <li><a href="#"> <img src="./images/js.png" alt=""><span class="js">JavaScript</span> </a></li>
+                <li><a href="#"><img src="./images/html.png" alt=""><span>HTML</span></a></li>
+                <li><a href="#"><img src="./images/css.png" alt=""><span>CSS</span></a></li>
+              </ul>
+            </li>
+            <li class="my-work">
+              <span >Frameworks</span>
```
Comment (sja-thedude) [inline index.html:240]: - [ ] Kindly make sure to follow the Figma design for the languages lists for the mobile version.
- [ ] Kindly make sure to follow the correct background image for the contact section of the mobile version as per the Figma design.

https://user-images.githubusercontent.com/70269507/165738524-7a012206-e138-4795-8cc2-0be15a272f8a.mov

File: index.html [hunk 3]
```diff
@@ -10,23 +10,24 @@
...
+    <title>Desktop version Portfolio</title>
   </head>
   <body>
     <header class="header">
       <nav class="top-nav">
         <div class="logo-placeholder">
           <a href="#"><span class="my-logo">My logo</span></a>
         </div>
-        <div class="normal-button">
-          <a href="#"><img src="./images/Union.svg" alt="more"></a>
```
Comment (sja-thedude) [inline index.html:22]: - [ ] Kindly avoid removing this code as it's for the mobile version to display the hamburger. You have to disable this in the desktop version with CSS.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @ndikumanaisaie, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good git-flow
✔️ Professional Readme
✔️ Good job on implementing the desktop version so far

## Required 
- [ ] Kindly edit your readme by adding your contact information and social media links and also a description for the project as it shows who worked on the repo and what you have worked on.
- [ ] It would be better to replace your Pull request title as it looks like a sentence. The Pull Request title should be in words like " Portfolio:  Desktop Version ".
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ndikumanaisaie** [comment]
Hi!

Thank you for your CR,

Please tell me if i have to duplicate code to add mobile images, content
and backgrounds because they are different from that of desktop version.

**Comment by sja-thedude** [comment]
> Hi!
> 
> Thank you for your CR,
> 
> Please tell me if i have to duplicate code to add mobile images, content
> and backgrounds because they are different from that of desktop version.

Hi @ndikumanaisaie 
You have to use the same html code and create the desktop and mobile version from it. You have to use css to display elements for both.
The elements which are not present in a specific version you have to use display: none for it. Which will make that specific part disappear. Example: For hamburger to disappear on desktop you have to use display: none inside the media query.

Basically you have to layout the elements with respect to the version with css.

**Comment by ndikumanaisaie** [comment]
Thank you for the clarification. But how can i change texts content and
images using CSS only without adding extra html code. For background images
i can see that it is possible

**Comment by ndikumanaisaie** [comment]
Hallo! Thank you. I tried hard and finaly i got it.

**Comment by Lucas-Masaba** [comment]
Thank you for taking the time to implement that 😎
